src/nupic/regions/sdr_classifier_region.py

```diff
@@ -57,17 +57,26 @@ class SDRClassifierRegion(PyRegion):
   During learning, the connection weights between input units and output units
   are adjusted to maximize the likelihood of the model
 
-
   The caller can choose to tell the region that the classifications for
   iteration N+K should be aligned with the activationPattern for iteration N.
   This results in the classifier producing predictions for K steps in advance.
   Any number of different K's can be specified, allowing the classifier to learn
   and infer multi-step predictions for a number of steps in advance.
+  
+  :param steps: (int) default=1,
+  :param alpha: (float) default=0.001,
+  :param verbosity: (int) default=0,
+  :param implementation: (string) default=None,
+  :param maxCategoryCount: (int) default=None
```

Comment:
@scottpurdy or @ywcui1990 Would be nice to fill in these param descriptions, I'm not sure what they are.